docker/rootfs/replace_config.py

```diff
@@ -10,8 +10,11 @@
 filedata = filedata.replace("prom2teamsgroupalertsby", os.environ.get("PROM2TEAMS_GROUP_ALERTS_BY"))
 filedata = filedata.replace("prom2teamslogslevel", os.environ.get("PROM2TEAMS_LOGLEVEL"))
 
-with open('/opt/prom2teams/config.ini', 'w') as file:
-  file.write(filedata)
+try:
+  with open('/opt/prom2teams/config.ini', 'w') as file:
+    file.write(filedata)
+except (PermissionError, OSError) as error:
+    pass
```

Comment:
Why are you bypassing error handling ? If there is actual error, why we ignore it

Reply:
Change to tmp config files in case of config file from docker volume.